tools/bls-signature-creator/.goreleaser.yml

```diff
@@ -0,0 +1,64 @@
+version: 1
```

Comment:
Simply adding this file won't add this to our releases page. If we want to release this as a tool, we should add this in the releaser workflow as well.